src/components/Pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Header } from '../../Header/Header';
+import { SideBar, SideBarProps } from '../../SideBar/SideBar';
+
+export const MainPage: React.VFC = ({lessonCardsList}: SideBarProps) => {
+    return (
+        <><Header /><SideBar lessonCardsList={lessonCardsList}/></>
```

Comment:
Some comments explaining what's happening might be nice:
i.e. `// This component takes in a list of lesson cards and shows them in the left sidebar of the main page`